src/Adapter/Order/CommandHandler/AddProductToOrderHandler.php

```diff
@@ -398,6 +406,21 @@ private function createSpecificPriceIfNeeded(
             $order->{Configuration::get('PS_TAX_ADDRESS_TYPE', null, null, $order->id_shop)}
         );
 
+        if (SpecificPrice::exists(
```

Comment:
What of specific price already existed but we want to change it? 🤔

Reply:
@sowbiba maybe you can help with this topic?

Reply:
with https://github.com/PrestaShop/PrestaShop/pull/19698/commits/7147d973b2ad07e075bd1e2ee2406f31e9490b7f I've removed "restoring whole order specific prices". It looks like its working well, but as @jolelievre said, it might be necessary for some upcoming order computations.
So idk, @sowbiba you are the one to tell us what could go wrong :sweat_smile:

Reply:
@MatShir is off this week so I'll try to help. 
I don't know much the topic, so let me know if you want to discuss more in depth about it.
But from what i can read, I would rather choose not to add new features as we are close to the final release. Can you confirm that not being able to change the specific price is not a regression ?

Reply:
This can be useful
https://github.com/PrestaShop/PrestaShop/issues/19595#issuecomment-642730057